samples/resources-gallery/build.gradle.kts

```diff
@@ -13,9 +13,9 @@ buildscript {
     }
     dependencies {
         classpath(moko.resourcesGradlePlugin)
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.8.10")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.9.0")
+        classpath("org.jetbrains.compose:compose-gradle-plugin:1.5.0")
```

Comment:
@Lobynya set kotlin 1.8.10 and compose 1.4.0 to test backward compatibility
but in resources-gallery-compose - set kotlin 1.9.0 and compose 1.5.0